charts/discovery-streams/values.yaml

```diff
@@ -10,6 +10,8 @@ global:
   ingress:
     dnsZone: ""
     rootDnsZone: ""
+  auth:
+    raptor_url: "http://raptor.admin/api/authorize"
```

Comment:
See above

Reply:
Updated in deploy https://github.com/SmartColumbusOS/discovery-streams-deploy/pull/43